src/typings.ts

```diff
@@ -15,19 +25,71 @@ export const supportedAuthnMethod = ['passwordless', 'sso', 'credentials'] as co
 export type AuthnMethod = typeof supportedAuthnMethod[number];
 
 export interface ConfigsOptions extends PrivateConfigsOptions {
+  /**
+   * Email used to pre-fill the email field in login or sign up or form.
+   *
+   * Set the default value via {@link MtLinkSdk.init}
+   */
   email?: string;
+  /**
+   * A redirection URL for redirecting a guest back to in the following condition:
```

Comment:
"condition" -> "conditions"
or
"condition" -> "one of the following conditions"